src/index.ts

```diff
@@ -163,6 +164,7 @@ program
   .option('--use_cairo_abi', 'Use the cairo abi instead of solidity for the inputs.', false)
   .option('--network <network>', 'Starknet network URL', process.env.STARKNET_NETWORK)
   .option('--no_wallet', 'Do not use a wallet for deployment.', false)
+  .option('--wallet <wallet>', 'Waller provider to use', process.env.STARKNET_WALLET)
```

Comment:
Shouldn't the description start with Wallet instead of Waller?

Reply:
Ah right, thanks for catching this!